src/app/demo/demo.component.html

```diff
@@ -209,11 +209,46 @@
   </div>
 
   <div class="fullscreen">
-    <div>
+    <nav class="navbar navbar-expand flex-column flex-md-row flex-wrap bd-navbar mb-4">
+      <a class="navbar-brand" href="#">
+          <img src="https://angular.io/assets/images/logos/angular/angular.svg" width="32px" height="32px"/>
+          @madoar/angular-archwizard
+      </a>
+      <button class="navbar-toggler"
+              type="button"
+              data-toggle="collapse"
+              data-target="#navbarsExampleDefault"
+              aria-controls="navbarsExampleDefault"
+              aria-expanded="false"
+              aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse">
+          <div dropdown class="btn-group ml-2 mr-2 mr-md-0">
+              <button class="btn btn-outline-secondary btn-sm text-uppercase dropdown-toggle" style="width: 60px;"
+                      dropdownToggle>{{dir}}</button>
+              <div *dropdownMenu class="dropdown-menu">
+                  <button (click)="changeDirTo('ltr')" class="dropdown-item btn-sm text-uppercase">ltr</button>
+                  <button (click)="changeDirTo('rtl')" class="dropdown-item btn-sm text-uppercase">rtl</button>
+              </div>
+          </div>
```

Comment:
By adding this code to `demo.component.html` you automatically add it to all examples. I'm not sure whether we want to add the `rtl`/`ltr` selection mechanism to all examples or if we want to have a single dedicated example for this functionality?

Reply:
IMHO I prefer it to affect all examples to show the users how rtl will look like in any existing example.
and as a way to consider rtl support in any upcoming feature example :D .

Reply:
Fair enough